### PR TITLE
Serhii rubayko less5

### PR DESCRIPTION
**Pull request by SkyLineR1001**
I'm proud of my recursive method for sum prime number, although maybe it's dareint))).

### CODE REVIEW

**Comment by SteranOo** [comment]
Dont spam with useless commits

**Comment by denys-kniaziev** [comment]
Create normal names for commits

File: Serhii Rubayko/Lesson5.Homework/Program.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    {
+        if (x==0) return y;
+        while (y != 0)
+        {
+            int tmp1;
+            if (x > y)
+            {
+                tmp1 = x % y;
+            }
+            else
```
Comment (denys-kniaziev) [inline Serhii Rubayko/Lesson5.Homework/Program.cs:17]: you can remove this, if y > x values will swap and no need to extra checks

File: Serhii Rubayko/Lesson5.Homework/Program.cs [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        return true;
+    }
+
+    // Common method 
+    static int PrimeNumSum(int z)
+    {
+        int result = 2;
+        for (int tmp = 3; tmp <= z; tmp++)
+        {
+           if (!IsPrime(tmp)) continue;
```
Comment (denys-kniaziev) [inline Serhii Rubayko/Lesson5.Homework/Program.cs:52]: why not?

```
if (IsPrime(tmp))
    result += tmp;
```

**Review by denys-kniaziev — APPROVED** [review]

**Pull request merged**